frontend/gulpfile.js

```diff
@@ -29,7 +29,7 @@ gulp.task('scripts:watch', ['scripts'], function () {
 
 gulp.task('images', function() {
```

Comment:
This task should be included in the `build` task below so that you don't need to run `collectstatic` for local development. Be sure and update the README since it currently instructs you to do so.

Reply:
done